packages/react/src/slider/root/useSliderRoot.ts

```diff
@@ -535,34 +574,27 @@ export namespace useSliderRoot {
   }
 
   export interface ReturnValue {
-    getRootProps: (
-      externalProps?: React.ComponentPropsWithRef<'div'>,
-    ) => React.ComponentPropsWithRef<'div'>;
     /**
      * The index of the active thumb.
      */
     active: number;
     'aria-labelledby'?: string;
-    handleInputChange: (
-      valueInput: number,
-      index: number,
-      event: React.KeyboardEvent | React.ChangeEvent,
-    ) => void;
+    /**
+     * Callback fired when drag ends and invokes onValueCommitted.
```

Comment:
nit: it's really up to the consumer to decide when this function is really called (as useSliderRoot doesn't call it itself as far as I can see). IMO a better description would be "Function to be called when drag ends. Invokes `onValueCommitted`"